barretenberg/cpp/src/barretenberg/translator_vm/translator_circuit_builder.test.cpp

```diff
@@ -58,7 +58,7 @@ TEST(TranslatorCircuitBuilder, CircuitBuilderBaseCase)
     Fq v = Fq::random_element();
     Fq x = Fq::random_element();
 
-    Fq previous_accumulator = Fq::random_element();
+    Fq previous_accumulator = Fq(0);
```

Comment:
the circuit checker now ensures that the accumulator starts at 0 so this test had to be modified